src/client-agent/agentd.c

```diff
@@ -154,6 +134,26 @@ void AgentdStart(int uid, int gid, const char *user, const char *group)
         }
     }
 
+    // Resolve hostnames
+    rc = 0;
+    while (rc < agt->server_count) {
+        if (OS_IsValidIP(agt->server[rc].rip, NULL) != 1) {
+            mdebug2("Resolving server hostname: %s", agt->server[rc].rip);
+            resolve_hostname(&agt->server[rc].rip, 5);
+            int rip_l = strlen(agt->server[rc].rip);
+            mdebug2("Server hostname resolved: %.*s", agt->server[rc].rip[rip_l - 1] == '/' ? rip_l - 1 : rip_l, agt->server[rc].rip);
+        }
+        rc++;
+    }
+
+    /* Connect remote */
+    rc = 0;
+    while (rc < agt->server_count) {
```

Comment:
This loop only has the functionality of logging the list of addresses "**Server IP Address:**". I think it will be more clear if we just print it together with the debug message "**Server hostname resolved:**" above.
We can also merge these two messages. into one